instructors.js

```diff
@@ -0,0 +1,49 @@
+const Teachers = [{
+  teacherImage: './images/professor1.jpg',
+  teacherName: 'Max Fischer',
+  teacherDomain: 'Web Developer in XYZ Company',
+  teacherAbout: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+}, {
+  teacherImage: './images/professor2.jpg',
+  teacherName: 'Vini Dawid',
+  teacherDomain: 'Web Developer in XYZ Company',
+  teacherAbout: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+}, {
+  teacherImage: './images/professor3.jpg',
+  teacherName: 'Ariana D Souza',
+  teacherDomain: 'Web Developer in XYZ Company',
+  teacherAbout: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+}, {
+  teacherImage: './images/professor4.jpg',
+  teacherName: 'Mikhail Nilov',
+  teacherDomain: 'Web Developer in XYZ Company',
+  teacherAbout: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
```

Comment:
[Optional] You seem to have some variation in the naming and usage of the image directory. You have `Image` and use `image` while providing src for some images.

I would suggest that you get consistent with file and directory names to avoid incorrect display in scenarios you might not expect.

<img width="871" alt="image" src="https://user-images.githubusercontent.com/20554583/197352739-77b4857b-c900-42f7-a815-314168ebfe5d.png">